### PR TITLE
Provide test code for wskdeploy sample project

### PR DESCRIPTION
**Pull request by upgle**
## Description
This PR adds a test code for wskdeploy sample project

### Changes
- All test cases use a test-fixtures folder as the workspace.
- Mocha version is upgraded (6.2.2 -> 7.2.0)
- Chai dependency is added.
- Sinon dependency is added to replace vscode API behavior.

### CODE REVIEW

File: src/test/suite/manifestList.test.ts [hunk 1]
```diff
@@ -55,48 +52,12 @@ const initState = {
...
-
-        assert.strictEqual(manifest.label, 'test1.yml');
-    });
-
-    after(() => {
-        if (vscode.workspace.workspaceFolders) {
-            fs.unlinkSync(resolve(vscode.workspace.workspaceFolders[0].uri.path, 'test0.yml'));
-            fs.unlinkSync(resolve(vscode.workspace.workspaceFolders[0].uri.path, 'test1.yml'));
-        }
+        assert.strictEqual(manifest.label, 'valid-manifest.yaml');
```
Comment (style95) [inline src/test/suite/manifestList.test.ts:61]: is `invalid-manifest.yaml` being used?

File: wskdeploy-template/minimal/src/package.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (style95) [inline wskdeploy-template/minimal/src/package.json:1]: Should we add any guides about how to use this plugin somewhere?
How to create a basic template, how to run a test, basic architecture, and so on?
I am just asking for your opinion.

File: src/test/suite/manifestList.test.ts (hunk 1 above)
Comment (upgle) [reply src/test/suite/manifestList.test.ts:61]: Yes, the `invalid-manifest.yaml` file should not be found by the manifest provider.

**Review by style95 — APPROVED** [review]
LGTM!

**Pull request merged**